cmake/sample_defs/sample_mission_cfg.h

```diff
@@ -47,6 +47,7 @@
 **      messages sent. If the pkt length field indicates the message is larger
 **      than this define, SB sends an event and rejects the send.
 **
+**      This length does not need to include an extra character for NULL termination.
```

Comment:
I'd remove this one to avoid confusion, the CFE_MISSION_SB_MAX_SB_MSG_SIZE isn't used for your classic "string".